products/firebase/terraform.yaml

```diff
@@ -25,13 +25,29 @@ overrides: !ruby/object:Overrides::ResourceOverrides
     examples:
       - !ruby/object:Provider::Terraform::Examples
         name: "firebase_project_basic"
-        min_version: 'beta'
+        min_version: "beta"
         primary_resource_id: "basic"
-        vars:
-          instance_name: "memory-cache"
         test_env_vars:
           org_id: :ORG_ID
-
+      - !ruby/object:Provider::Terraform::Examples
+        name: "firebase_project_full"
+        min_version: "beta"
+        primary_resource_id: "basic"
```

Comment:
`full` is a better resource id for this test.

Reply:
The example in the original was called basic so I didn't rename it, but I'll rename it and change the code reference as you suggest.

Reply:
This comment was for the original PR and it's changed so now the "basic" name actually does make sense :)